pages/builders-challenge/index.html

```diff
@@ -56,7 +56,7 @@ <h2>by Mozilla Builders</h2>
 <section class="mzp-l-content mzp-t-content-xl c-challenge-event">
     <p class="c-event-tagline">AI is changing everything. Now is the time to develop it responsibly. <br> Join us in the Responsible AI Challenge and help us shape the future of AI.</p>
     <p class="c-event-date">In-person event: May 31, 2023 in San Francisco, CA</p>
-    <p>Applications have closed. Thank you for your interest and participation</p>
+    <img src="/img/builders-challenge/AI-Challenge-schedule.png"></img>
```

Comment:
```suggestion
    <img src="/img/builders-challenge/AI-Challenge-schedule.png">
```
Also missing the required `alt` attribute.

Reply:
Using an image, especially one with no alt text explanation is not accessible. This isn't a ton of info. Can we code this into a table on the page so people using screen readers can also access the information?

Reply:
Or maybe the image could be a link to a properly made PDF?